_episodes/05-definition-files.md

```diff
@@ -32,10 +32,11 @@ The following recipe shows how to build a hello-world container, and run the con
   From: ubuntu:20.04
 
   %runscript
-  echo "Hello World"
+    echo "Hello World"
```

Comment:
There's now spaces in front of `echo` that should probably be removed

Reply:
Yeah thanks for the comment. The space is intentional (based on the style used in this lesson and in other documentation). I had missed the space earlier.